packages/salesforcedx-vscode-core/src/commands/forceApexTestRunCodeAction.ts

```diff
@@ -60,31 +61,41 @@ const forceApexTestRunCacheService = ForceApexTestRunCacheService.getInstance();
 // build force:apex:test:run w/ given test class or test method
 export class ForceApexTestRunCodeActionExecutor extends SfdxCommandletExecutor<{}> {
   private test: string;
+  private shouldGetCodeCoverage: boolean = false;
+  private builder: SfdxCommandBuilder = new SfdxCommandBuilder();
 
-  public constructor(test: string) {
+  public constructor(test: string, shouldGetCodeCoverage: boolean) {
     super();
     this.test = test || '';
+    this.shouldGetCodeCoverage = shouldGetCodeCoverage;
   }
 
   public build(data: {}): Command {
-    return new SfdxCommandBuilder()
+    this.builder = this.builder
       .withDescription(
         nls.localize('force_apex_test_run_codeAction_description_text')
       )
       .withArg('force:apex:test:run')
       .withFlag('--tests', this.test)
       .withFlag('--resultformat', 'human')
       .withArg('--synchronous')
-      .withFlag('--loglevel', 'error')
-      .build();
+      .withFlag('--loglevel', 'error');
+
+    if (this.shouldGetCodeCoverage) {
+      this.builder = this.builder.withArg('--codecoverage');
+    }
+
+    return this.builder.build();
   }
 }
 
 async function forceApexTestRunCodeAction(test: string) {
+  const getCodeCoverage: boolean = SfdxCoreSettings.getInstance()
```

Comment:
Same here. Get it from the exported singleton at https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-core/src/settings/index.ts#L10